tests/integration/lag/01-l3-lag.yml

```diff
@@ -1,25 +1,25 @@
 message: |
   The device under is a router with a L3 LAG link connected to a FRR device. The
-  devices should be able to establish an OSPF adjacency.
-
-groups:
-  switches:
-    members: [dut, xr]
-    module: [lag, ospf]
+  FRR device should be able to ping the loopback interface of DUT.
 
 nodes:
   dut:
+    module: [ lag ]
+    role: router
   xr:
     device: frr
+    module: [ lag, routing ]
+    routing.static:
+    - ipv4: 0.0.0.0/0
+      nexthop.node: dut
 
 links:
 - lag.members: [dut-xr, dut-xr]
   mtu: 1500
 
 validate:
-  adj:
-    description: Check OSPF adjacencies
-    wait_msg: Waiting for OSPF adjacency process to complete
-    wait: 50
-    nodes: [xr]
-    plugin: ospf_neighbor(nodes.dut.ospf.router_id)
+  ping:
+    wait_msg: Waiting for LAG to be established
+    wait: 15
+    nodes: [ xr ]
+    plugin: ping(nodes.dut.loopback.ipv4)
```

Comment:
Could set MTU to 1600 and do a ping of size 1600, to verify that the device configures MTU correctly for the LAG interface

OSPF was testing this for us before, but not anymore

Reply:
Thank you. Will do (I'll probably creatively reuse the initial MTU test)